kernel/spl/BUILD

```diff
@@ -1,14 +1,25 @@
-OPTS+=" --with-linux=$BUILD_DIRECTORY/linux \
-        --with-linux-obj=$BUILD_DIRECTORY/linux \
-        --with-config=all"
+./autogen.sh &&
+
+OPTS+=" --disable-static  \
+        --with-config=all \
+        --with-linux=$BUILD_DIRECTORY/linux \
+        --with-linux-obj=$BUILD_DIRECTORY/linux" &&
 
 if [ "$(arch)" == "i686" ]; then
   OPTS+=" --enable-atomic-spinlocks"
-fi 
+fi &&
 
 default_config &&
 make &&
-make DESTDIR=$SOURCE_DIRECTORY/__pkg install &&
 prepare_install &&
+
+devoke_installwatch &&
+
+mkdir __pkg &&
+make DESTDIR=$SOURCE_DIRECTORY/__pkg install &&
 cd $SOURCE_DIRECTORY/__pkg &&
+mv lib usr/ &&
```

Comment:
@Florin65 this actually break the installation since the kernel modules will be installed in the wrong location. I know that you are using an /usr installation, the correct way would be to have /lib -> /usr/lib on your system.